opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java

```diff
@@ -479,8 +502,8 @@ private void startForm(Feature feature, CardDetails cardDetails, String encounte
             } else if (BuildConfig.BUILD_COUNTRY == Country.ZAMBIA) {
                 formName = SPRAY_FORM;
             }
-            String sprayStatus = cardDetails == null ? null : cardDetails.getSprayStatus();
-            String familyHead = cardDetails == null ? null : cardDetails.getFamilyHead();
+            String sprayStatus = cardDetails == null ? null : cardDetails.getStatus();
+            String familyHead = cardDetails == null ? null : ((SprayCardDetails) cardDetails).getFamilyHead();
```

Comment:
Add a check that `cardDetails` instance of `SprayCardDetails`, otherwise a runtime error can occur if not able to cast

Reply:
Done.